### DISCUSSION TITLE
Cover image issues

### DISCUSSION THREAD

**Discussion by chartgerink** [avitorio/outstatic#82] [Q&A]
Hi 👋 Loving outstatic!

I ran into some issues for the cover image when uploaded:

1. By link
2. As a file

For (1), I get this error:
<img width="992" alt="Screenshot 2023-01-04 at 14 51 19" src="https://user-images.githubusercontent.com/2946344/210569518-b933ce10-a918-4b98-a79f-d1ddc9a7c1e6.png">

I know how to resolve it manually in Next, but the people who will manage the CMS won't.

For (2), I try to upload a file but nothing seems to happen.

Anybody else have this problem?

**Comment by avitorio** [comment]
Thanks for these @chartgerink - I'll investigate them asap.

**Reply by ZiyadBoshima** [reply]
I am having a similar issue where I can't upload an image in the Cover Image field. Do I have to first place the image in public/images and then upload? Or should I be able to upload from anywhere?

**Reply by avitorio** [reply]
Hey,

If you arenuploading from a link/url, you may need to add the domain of the image URL to your config file:

https://nextjs.org/docs/app/api-reference/components/image

If you upload from a file it should just work.

**Reply by ZiyadBoshima** [reply]
Hey,

thank you for your response. Uploading using a link works fine. Uploading a file directly will trigger an error on the webpage for a split second: "The image failed to load, try submitting again."

I have tried submitting different images (different image type too) from different directories, but nothing seemed to work. The console does not seem to show any problems.

**Reply by avitorio** [reply]
Weird. Can you provide a minimal repository with this error?

**Reply by ZiyadBoshima** [reply]
You can check out my "blog" repo. It is still at an early stage of development, and has this error.

[link](https://github.com/ZiyadBoshima/blog)

**Reply by avitorio** [reply]
Thanks, I'll try to review this in the next few days.

**Reply by avitorio** [reply]
Hey @ZiyadBoshima,
I cloned your repo and tested locally. I can upload local images to Cover Images...
Can you create an issue and tell me exactly the steps you are taking? 
Are you using npm, yarn, pnpm?

**Reply by ZiyadBoshima** [reply]
Hey @avitorio,
thank you for your response, and for your time. I figured out what the problem was. Apparently, there is a compatibility problem of Outstatic with Safari, as that is the browser I use. I just tried to upload a cover image using chrome as the browser, and it worked fine!

**Reply by avitorio** [reply]
Oh wow, thanks for the update!
Can you create an issue for this?

**Reply by ZiyadBoshima** [reply]
Of course!
